### PR TITLE
Alters 'time difference' to be listed in seconds (with decimals).

### PR DESCRIPTION
**Pull request by Qqwy**
This PR has a very small scope:

It alters the 'time difference: ...' text to list the time difference in seconds (with the milliseconds as decimals) rather than the raw milliseconds. :slightly_smiling_face: 

I made this one because it was nice 'low hanging fruit' to get my feets wet.

### CODE REVIEW

File: src/com/tinytimrob/ppse/nmo/MainDialog.java [hunk 1]
```diff
@@ -1647,7 +1647,8 @@ protected void tick()
 		{
 			resetActivityTimer("pause");
 		}
-		long timeDiff = paused ? 0 : (now - lastActivityTime);
+		long timeDiff = paused ? 0 : Math.abs(now - lastActivityTime);
+		timeDiff = Math.abs(timeDiff); // Normalizes negative zero to positive zero.
```
Comment (Tinytimrob) [inline src/com/tinytimrob/ppse/nmo/MainDialog.java]: Why use Math.abs? long type doesn't have a negative zero.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+syntax: glob
```
Comment (Tinytimrob) [inline .gitignore]: syntax: glob isn't required for gitignore.

**Comment by Qqwy** [comment]
I actually think `Math.abs` is indeed the wrong approach. Now that I am somewhat more awake, I am aware of what is actually going on: `now - lastActivityTime` might return a slightly negative number
(I expect that `lastActivityTime` is read out somewhat later than this frame's `now` ?).

A better approach might be to clamp `timeDiff` at 0. (And possibly change its type to an unsigned long?) What do you think?

The change that has less impact on the existing code is of course to move the `Math.abs` statement to line 1691, where `humanlyReadableTimeDiff` is defined, which is what I've implemented in the version that is now available in this Pull Request.

**Comment by Tinytimrob** [comment]
I had a look into the problem and it seems to be a timing issue. The activity timer is updated by some of the input integrations asynchronously to the main thread. Because the value of `now` is constant throughout the main thread tick this is sometimes causing `lastActivityTime` to be greater than `now` which results in `timeDiff` having a negative value.

I'll accept your workaround for now because I think that it's the least messy option but I'll probably revisit this issue at a later date

**Pull request merged**